rosbag2_cpp/src/rosbag2_cpp/typesupport_helpers.cpp

```diff
@@ -105,33 +106,51 @@ get_typesupport(const std::string & type, const std::string & typesupport_identi
   std::string type_name;
   std::tie(package_name, middle_module, type_name) = extract_type_identifier(type);
 
-  std::string poco_dynamic_loading_error = "Something went wrong loading the typesupport library "
-    "for message type " + package_name + "/" + type_name + ".";
+  std::stringstream rcutils_dynamic_loading_error;
+  rcutils_dynamic_loading_error <<
+    "Something went wrong loading the typesupport library for message type " << package_name <<
+    "/" << type_name << ".";
```

Comment:
why is this change necessary?

Reply:
It was a suggestion from @piraka9011 

https://github.com/ros2/rosbag2/pull/322#discussion_r393258319

Reply:
@Karsten1987 It's a convention we've been using for constructing error strings in other packages.
No strong opinions but prefer a single convention.